vdev/src/testing/integration.rs

```diff
@@ -179,7 +179,10 @@ impl Compose {
                 // Inject the networks block
                 config.networks.insert(
                     "default".to_string(),
-                    BTreeMap::from_iter([("name".to_string(), network.clone())]),
+                    BTreeMap::from_iter([
+                        ("name".to_string(), network.clone()),
+                        ("external".to_string(), "true".to_string()),
```

Comment:
`external: true` was required to get the integration tests running after I upgraded Docker locally. I'll verify this doesn't break anything in CI.